forge/test/operators/pytorch/eltwise_unary/test_unary.py

```diff
@@ -234,6 +248,63 @@ class TestCollectionData:
             math_fidelities=[MathFidelity.HiFi4],
             failing_reason=FailingReasons.DATA_MISMATCH,
         ),
+        TestCollection(
+            operators=["abs", "cos", "neg", "sin"],
+            input_sources=[InputSource.FROM_HOST],
+            input_shapes=[(1, 2, 3, 4)],
+            dev_data_formats=[
+                DataFormat.Int8,
+                DataFormat.Int32,
+            ],
+            math_fidelities=[MathFidelity.HiFi4],
+            failing_reason=FailingReasons.DATA_MISMATCH,
+        ),
+        TestCollection(
+            operators=["rsqrt"],
+            input_sources=[InputSource.FROM_HOST],
+            input_shapes=[(1, 2, 3, 4)],
+            dev_data_formats=[
```

Comment:
How do we construct these sets of dev_data_formats based on op?

Reply:
We use all data formats with fixed values for the input source, input shape, and math fidelity, and we test data formats for all operators. This one TestCollection is a part of failing rules - tests for testing data formats fail for these data format values.